application/src/main/java/org/thingsboard/server/service/install/SqlDatabaseUpgradeService.java

```diff
@@ -480,7 +488,41 @@ public void upgradeDatabase(String fromVersion) throws Exception {
                     } catch (Exception e) {
                         log.error("Failed updating schema!!!", e);
                     }
+                  
+                    deviceProfileRepository.findAll().forEach(deviceProfile -> {
+                        if(deviceProfile.getProfileData().has("alarms")) {
+                            JsonNode array = deviceProfile.getProfileData().get("alarms");
+                            for (JsonNode node : array) {
+                                if (node.has("createRules")) {
+                                    JsonNode createRules = node.get("createRules");
```

Comment:
You can iterate through all the keys. OR at least use Severity enum. This is terrible code duplication.

Reply:
What if you missed one of the severities?